### ISSUE TITLE
gRPC: Toggle Message requests when toggling between interfaces

### ISSUE THREAD

**Issue by vikiea** [a85/POSTMan-Chrome-Extension#11183]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

Currently, when I switch between interfaces under the same proto, the content of the message body remains the same. 

### Describe the solution you'd like

I want it to switch with the interface and save the current message body when it switches so it can be used the next time it switches back

**Comment by codenirvana** [comment]
@vikiea You can generate a new message after switching the method.

<img width="1056" alt="Screenshot 2022-08-12 at 12 31 38 PM" src="https://user-images.githubusercontent.com/4865836/184301820-91d9d3a1-32fb-481e-885d-da6a308b51c9.png">

If I understood your use case correctly, you are expecting it to automatically re-generate a new example message on method switch, right?

**Comment by vikiea** [comment]
> 

not really, I want to regenerate the previously saved message body because the generated sample message does not meet my requirements

**Comment by shamasis** [comment]
@vikiea we are working on being able to save your requests as examples that you can use from a quick dropdown menu beside the "generate example message" button.

Something like this 
<img width="664" alt="image" src="https://user-images.githubusercontent.com/232373/189293879-4e9268c6-f0bf-48a1-9858-d15f6de4603c.png">

**Comment by vikiea** [comment]
nice work!
